lib/datadog/tracing/span_event.rb

```diff
@@ -69,7 +69,11 @@ def to_native_format
 
       private
 
-      MIN_INT64_SIGNED = -2**63
+      # Steep: `Integer ** Integer` can also result in Rational (e.g. 10**-2)
+      # which is why it gives an error if MIN_INT64_SIGNED is set to -2**63.
+      MIN_INT64_SIGNED = -2 << 62
+      # DEV: This is incorrect, 2 << 63 - 1 is not the same as 2**63 - 1 (offset by 1).
+      # The correct formula is `(2 << 62) - 1`. Check if it is a breaking change before fixing it.
       MAX_INT64_SIGNED = 2 << 63 - 1
```

Comment:
This is a nice catch, and indeed it's not correct -- there is an off-by-one here.

I think it's very much worth fixing, either on this PR or another; let's not just leave a comment here and move on ;)